tests/TestCase.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests;
+
+use PHPUnit\Framework\TestCase as BaseTestCase;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ */
+abstract class TestCase extends BaseTestCase
+{
+}
```

Comment:
what is the added value over not using `PHPUnit\Framework\TestCase` directly?

Reply:
I'm curious too.
As a Contributor if I see `extends \PHPUnit\Framework\TestCase` I'm already in a confortable zone where I know what is going on; if I notice a project-specific base test case I must open the abstract class to inspect eventual setUp/tearDown overrides etc: it's and added step, useless for now.
But maybe you have something in mind for the future.

Reply:
to do this https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3285